commit e8dfacf141844ec1c2c78ecc98d85b3698bf946c
Author: Nam T. Nguyen <namnguyen@chromium.org>
Date:   Thu Nov 13 09:12:47 2014 -0800

    vboot: Fix load_kernel_tests to call the right cgpt
    
    load_kernel_tests called out to "cgpt" as is and this is interpreted as
    "/usr/bin/cgpt" on the host. This CL fixes that by using the right
    version in "build/install_for_test/bin/cgpt".
    
    BUG=none
    BRANCH=none
    TEST=rename /usr/bin/cgpt; make runtests
    
    Change-Id: I65ee0f271b081da6f719764e22c14fef05cdb65b
    Reviewed-on: https://chromium-review.googlesource.com/229590
    Reviewed-by: Randall Spangler <rspangler@chromium.org>
    Commit-Queue: Nam Nguyen <namnguyen@chromium.org>
    Tested-by: Nam Nguyen <namnguyen@chromium.org>

diff --git a/tests/load_kernel_tests.sh b/tests/load_kernel_tests.sh
index c7c0dc1..c8422fd 100755
--- a/tests/load_kernel_tests.sh
+++ b/tests/load_kernel_tests.sh
@@ -11,6 +11,8 @@
 
 set -e
 
+CGPT=${BIN_DIR}/cgpt
+
 echo 'Creating test kernel'
 
 # Run tests in a dedicated directory for easy cleanup or debugging.
@@ -58,9 +60,9 @@ happy 'Kernel verification succeeded'
 # Now create a dummy disk image
 echo 'Creating test disk image'
 dd if=/dev/zero of=disk.test bs=1024 count=1024
-cgpt create disk.test
-cgpt add -i 1 -S 1 -P 1 -b 64 -s 960 -t kernel -l kernelA disk.test
-cgpt show disk.test
+${CGPT} create disk.test
+${CGPT} add -i 1 -S 1 -P 1 -b 64 -s 960 -t kernel -l kernelA disk.test
+${CGPT} show disk.test
 
 # And insert the kernel into it
 dd if=kernel.test of=disk.test bs=512 seek=64 conv=notrunc
